### ISSUE TITLE
Navigation Path (Breadcrumbs) render bug

### ISSUE THREAD

**Issue by adminradio** [squidfunk/mkdocs-material#4865]
### Bug description

I’m trying the new navigation paths feature, but stumbled over a visual hickup. The following mkdocs.ymlgenerates the output in the screenshot (the red line shows the problem):

```yaml
site_name: My Docs

theme:
  name: material
  features:
    - navigation.path

nav:
  - Start:
    - index.md
    - Level 1:
      - l1/index.md
      - Level 1.1: l1/l1.1/index.md
```
<img width="505" alt="Bildschirm­foto 2023-01-15 um 17 40 40" src="https://user-images.githubusercontent.com/2880143/212569080-8422a2c2-7cab-4f92-b421-70fc9071e148.png">

### Related links

- https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/?h=navigation+path#navigation-path

### Reproduction

[example.zip](https://github.com/squidfunk/mkdocs-material/files/10421337/example.zip)

### Steps to reproduce

1. init a new mkdocs project
2. add nested nav items with content on section item, like so:

```
nav:
  - Start:
    - index.md
    - Level 1:
      - l1/index.md
      - Level 1.1: l1/l1.1/index.md
```

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/bug-report/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) or [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/bug-report/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/bug-report/reproduction/).

**Comment by wilhelmer** [comment]
I can confirm this. The bug occurs whenever there's no topic at root level.

So this works:

```yml
nav:
  - Start: index.md # This is our root level topic
  - Level 1:
    - l1/index.md
    - Level 1.1: l1/l1.1/index.md
```

But this doesn't:

```yml
nav:
  - Start: # No root level topic here
    - index.md 
    - Level 1:
      - l1/index.md
      - Level 1.1: l1/l1.1/index.md
```

**Comment by squidfunk** [comment]
Thanks for reporting! Fixed in [`4ac3eea3d`](https://github.com/squidfunk/mkdocs-material-insiders/commit/4ac3eea3d151c8cdee28728d821127afc78654f2) 😊

**Comment by wilhelmer** [comment]
Works, thanks! 

Side note: I'd rather not have the breadcrumbs on level 1 pages at all, as in most cases, it will only duplicate the page title:

<img width="973" alt="image" src="https://user-images.githubusercontent.com/2234833/212654871-c70c0fe4-78e1-46cc-87a5-2c334722dca3.png">

I changed that by wrapping `page.html` in `{% if page.ancestors | length > 1 %}`. But maybe you'll want to add that as a general option.

**Comment by squidfunk** [comment]
Good idea, this has already been proposed in #4875.

**Comment by squidfunk** [comment]
Breadcrumbs are now only shown when the depth `>= 2`, and can be hidden on a per-page basis (or per section, by using the [built-in meta plugin](https://squidfunk.github.io/mkdocs-material/reference/#built-in-meta-plugin)). For more, see https://github.com/squidfunk/mkdocs-material/issues/4875#issuecomment-1384558862

**Comment by wilhelmer** [comment]
... and it's [live](https://docs.baslerweb.com/basler-ace-2-5gige) in our docs! 🚀 

Really great addition to Material, thanks.

**Comment by squidfunk** [comment]
Released as part of [9.0.5-insiders-4.28.1](https://squidfunk.github.io/mkdocs-material/insiders/changelog/#4.28.1).
